tools/homebrew-start/lanraragi

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+trap "kill 0" SIGINT
+export BREWMODE=1
+workdir=$(dirname $0)/$(dirname $(readlink $0))/..
+redis-server & npm start --prefix "${workdir}"
```

Comment:
I quickly tried out Homebrew for Linux and got the following error upon running the app:  

```
Can't locate Mojo/Base.pm in @INC (you may need to install the Mojo::Base module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0 /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0 /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.30.0) at ./script/launcher.pl line 7.
BEGIN failed--compilation aborted at ./script/launcher.pl line 7.
```  
It looks like your script does install Mojolicious properly, but since you're not using the cpanfile some dependencies might be missing?  

I'll try this on a real Mac tomorrow as it's the true target and I'm pretty sure it'll work there, just letting you know.

Reply:
Also, launching Redis this way doesn't shut the server down after Ctrl-C-ing out of the main app. Not sure how you could fix that(or if it's that much of a problem.)

Reply:
Indeed, Linux is not the target. Although Homebrew claims to work on Linux as well, I think solving this issue could be done by opening an issue over at my [tap](https://github.com/Bl4Cc4t/homebrew-other) :)

As for `redis`, it gets killed just fine for me. That’s what the first line should handle.